src/pykeen/pipeline/api.py

```diff
@@ -908,6 +919,15 @@ def pipeline(  # noqa: C901
         optimizer_kwargs,
         params=model_instance.get_grad_params(),
     )
+
+    lr_scheduler_instance = lr_scheduler_resolver.make(
+        lr_scheduler,
+        lr_scheduler_kwargs,
+        optimizer=optimizer_instance,
+    )
+
+    # Many changes to the optimizer by an optional lr scheduler are shown in the optimizer itself and thus logged
+    # TODO: Check whether this is sufficient or the lr scheduler should also have a result tracker
```

Comment:
@mberr 
Related to your remark https://github.com/pykeen/pykeen/pull/492#issuecomment-862524473

Reply:
does this one need to be figured out before merge?